lib/mix/tasks/igniter.upgrade_igniter.ex

```diff
@@ -60,7 +61,19 @@ defmodule Mix.Tasks.Igniter.UpgradeIgniter do
       arity: 1
     )
     |> Igniter.add_notice(
-      "Igniter.Code.Module.parse/1 was deprecated in favor of `Igniter.Project.Module.parse/1"
```

Comment:
This is highlighting weird, but the only change to this existing upgrade function is removing the stray backtick.